.env.example

```diff
@@ -7,3 +7,5 @@ POSTGRES_URI_BOT=sqlite:///
 
 # Variables for various command cogs.
 AOC_SESSION_ID=
+SB_BASE_THRESHOLD=8
+SB_BIG_THRESHOLD=24
```

Comment:
Not critical for PR: Can you add these into the Wiki page please https://github.com/UQComputingSociety/uqcsbot-discord/wiki/Tokens-and-Environment-Variables

Reply:
https://github.com/UQComputingSociety/uqcsbot-discord/wiki/Tokens-and-Environment-Variables#sb_base_threshold-sb_big_threshold